streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -17,40 +17,34 @@
 
 package org.apache.kafka.streams.processor.internals.assignment;
 
+
 import java.util.HashSet;
 import java.util.Set;
 
 public class ClientState<T> {
```

Comment:
@mjsax - i had the same thought and did start down the path of changing it. Soon realised it was a battle for another day as it would've muddied the PR